packages/web/src/components/list/utils.js

```diff
@@ -81,24 +81,27 @@ const getAggsQuery = (query, props) => {
 	return { ...clonedQuery, ...extractQueryFromDefaultQuery(props.defaultQuery) };
 };
 
-const getCompositeAggsQuery = (query, props, after) => {
+const getCompositeAggsQuery = (query, props, after, showTopHits = false) => {
```

Comment:
@ShahAnuj2610 Can you move the whole content of this file in `reactivecore` so we don't need to make these changes for `vue` & `react-native`.

Reply:
https://github.com/appbaseio/reactivecore/pull/48